fsw/src/eci_app_hk.h

```diff
@@ -36,7 +36,7 @@ extern "C"
     uint16 Latched[APP_FAULTREP_BITFIELD_WORDS];
 #endif
     /** Number of each SIL App-Subscribed Messages Received */
-    uint16 MsgRcvCnt[SIZEOF_ARRAY(ECI_MsgRcv) - 1];
+    uint16 MsgRcvCnt[SIZEOF_ARRAY(ECI_MsgRcv)];
```

Comment:
Is changing the size of this array fixing an unrelated bug? I can't see how this would be related to the CFS version update.

Reply:
Unrelated bug, zero size array error.